hugo/content/about.md

```diff
@@ -0,0 +1,58 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.
+
+## Source Code & Development
+
+The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).
+
+## Licenses
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a BSD-style [License](https://www.mumble.info/LICENSE).
+
+All Licenses are also listed in the GitHub repositories, see: [Main Mumble License](https://github.com/mumble-voip/mumble/blob/master/LICENSE) & [Overview of other Mumble Projects](https://github.com/mumble-voip/)
+
+
+# About us
+
+We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.
+
+## Core Team
+
+Active Team Members are:
+
+| Name: | Github Profile: | Assignment: |
+| --- | --- | --- |
+| Davide Beatrici | [davidebeatrici](https://github.com/davidebeatrici) | Core developer |
+| Robert Adam | [Krzmbrzl](https://github.com/Krzmbrzl) | Core developer, GitHub Management |
+| Nik Reist | [ZeroAbility](https://github.com/ZeroAbility) | Release Management |
+| Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |
+| - | [Natenom](https://github.com/Natenom) | Testing |
+
+
+## Contributors
+
+Contributions from the community are an important part of the project.
+You can find a [list of contributors on GitHub](https://github.com/mumble-voip/mumble/graphs/contributors).
+
+
+## Contact
+
+Take a look at our [Contact page](https://www.mumble.info/contact).
```

Comment:
This should be a relative hugo native link.

Reply:
I don't like it, but I will add it anyway.

Reply:
Do you have a reason for disliking it?

Not using native, relative links leads to technical issues if you make changes to how and where you deploy.

Reply:
I like and understand the concept of relative linking; what I dislike is described here: #135.
In the meantime I already added relref links in my offline version, which I will upload along with all other changes, once it's ready.